src/reference/asciidoc/jdbc.adoc

```diff
@@ -433,6 +433,8 @@ For this purpose, the `INT_CHANNEL_MESSAGE` has a `MESSAGE_PRIORITY` column to s
 In addition, a new `MESSAGE_SEQUENCE` column is also provided to achieve a robust first-in-first-out (FIFO) polling mechanism, even when multiple messages are stored with the same priority in the same millisecond.
 Messages are polled (selected) from the database with `order by MESSAGE_PRIORITY DESC NULLS LAST, CREATED_DATE, MESSAGE_SEQUENCE`.
 
+IMPORTANT: The `capacity` attribute is not allowed when the `message-store` attribute is used.
```

Comment:
??? I think you place this sentence in the JDBC chapter accidentally.
I'm sure you meant the 
```
 [[channel-configuration-prioritychannel]]
 ===== PriorityChannel Configuration
```
section in the `channel.adoc`